storage-proofs-porep/src/stacked/mod.rs

```diff
@@ -1,3 +1,5 @@
+pub use crate::encode::{decode, encode};
```

Comment:
To me it feels a bit strange to make things public from higher up the hierarchy. What about making the `mod encode` in `src/lib.rs` public instead?